client/api/src/main/java/org/opennms/velocloud/client/api/model/MetricsTraffic.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.velocloud.client.api.model;
+
+import java.util.Map;
+
+import com.google.common.base.MoreObjects;
+
+public class MetricsTraffic {
+
+    private final Map<String, Traffic> perApplication;
+    private final Map<String, Traffic> perSource;
+    private final Map<String, Traffic> perDestination;
+    private final Map<String, Traffic> perBusiness;
+
+    private MetricsTraffic(final MetricsTraffic.Builder builder) {
```

Comment:
Shouldn't we check for not null here?

Reply:
parameter `builder` can not be `null` because it is used only once in a build(). The field of `Builder` and also of `MetricsTraffic` can be `null` when no data provided is.